controller/src/test/java/io/pravega/controller/server/ZKBackedControllerServiceStarterTest.java

```diff
@@ -262,6 +262,7 @@ protected ControllerServiceConfig createControllerServiceConfigWithEventProcesso
                                                                                             .port(grpcPort)
                                                                                             .authorizationEnabled(false)
                                                                                             .tlsEnabled(false)
+                                                                                            .tlsProtocolVersion("TLSv1.2,TLSv1.3")
```

Comment:
Use the constant defined in SecurityConfigDefaults  here an all other places.

Reply:
Done. Thanks.